app/dashboard/models.py

```diff
@@ -279,6 +279,10 @@ def value_in_usdt(self):
         except Exception:
             return None
 
+    @property
+    def eth_value_in_usdt(self):
```

Comment:
itd be great to have a way to make the app aware of the conversion rate without attaching it to the model (since it's technically not a part of the model)

Reply:
I added as a property because I wasn't quite sure how to pipe this into the email. I will dig into that a bit more.

Reply:
Oh perfect. I will put it there then. I will also make this method dynamic so it will convert to the desired token instead of just ETH.

Reply:
@owocki Sorry, just getting back to this project now. So basically I will be capturing the current USD to ETH rate and then storing that in the context? What if it is a token other than ETH?

Reply:
good question.. maybe its easier to just keep this on the model so it's in the context of the base currency of the bounty

Reply:
Okay, sounds good. I am testing now and should have a commit soon.